streaming/src/main/scala/org/apache/spark/streaming/receiver/Receiver.scala

```diff
@@ -271,7 +271,7 @@ abstract class Receiver[T](val storageLevel: StorageLevel) extends Serializable
   }
 
   /** Get the attached executor. */
-  private def executor = {
+  private[streaming] def executor = {
```

Comment:
`PrivateTester` won't work here. It assumes the private method is declared in the type of the receiver. It only works if the *runtime* type of the receiver is the type that declares the private method. Not the case here, where the method is declared in `Receiver`, but the invocation is on a concrete subclass (`DummyReceiver`). Relevant code [here](https://github.com/scalatest/scalatest/blob/master/scalatest/src/main/scala/org/scalatest/PrivateMethodTester.scala#L243).

I could probably code something along those lines, but I think it's better to use qualified private. Are there any downsides to relying on language support for visibility? I doubt any user-code would declare things in `o.a.s.streaming`.